src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V2/DownloadedPackagesSQLLiteStoreV2.swift

```diff
@@ -546,8 +550,8 @@ private extension FMResultSet {
 extension DownloadedPackagesSQLLiteStoreV2 {
 	convenience init(fileName: String) {
 
-		let fileManager = FileManager()
-		guard let documentDir = fileManager.urls(for: .applicationSupportDirectory, in: .userDomainMask).first else {
+		let fileManager = FileManager.default
+		guard let documentDir = try? fileManager.url(for: .applicationSupportDirectory, in: .userDomainMask, appropriateFor: nil, create: true) else {
 			fatalError("unable to determine document dir")
```

Comment:
document directory perhaps?

Reply:
@naveeddotio For this kind of change please contact @whiskey

Reply:
@naveeddotio  the name is now misleading. It WAS the documents dir in the past and during my refactoring to app support I missed the variable name here.